doc/python/webgl-vs-svg.md

```diff
@@ -198,3 +207,99 @@ fig.show()
 ### Reference
 
 See https://plotly.com/python/reference/scattergl/ for more information and chart attribute options!
+
+## Pandas and NumPy for Improved Performance
+
+Improve the performance of generating Plotly figures that use a large number of data points by using NumPy arrays and other objects that can be converted to NumPy arrays, such as Pandas Series and Index objects.
+
+Plotly.py uses Plotly.js for rendering, which supports typed arrays. In Plotly.py, NumPy array and NumPy-convertible arrays are base64 encoded before being passed to Plotly.js for rendering.
+
+### Arrays and Data Types Supported
+
+The following types of objects in Python are supported for improved performance:
+
+- Numpy `numpy.ndarray` objects.
+- Pandas Index, `pandas.Index`, or Series, `pandas.Series`, objects.
+- Array objects that can be converted to `numpy.ndarray` objects. i.e., they implement `"__array__"` or `"__array_interface__"` and return a `numpy.ndarray`.
+
+The following array data types are supported:
+
+- int8
+- uint8
```

Comment:
It would be nice to mention the range of each these integer data types.

Reply:
I've added a link out to the numpy docs which show the ranges. I think that would be better than writing them out here.